farcy/__init__.py

```diff
@@ -49,6 +49,52 @@
 PR_ISSUE_COMMENT_FORMAT = '_{0}_ {{0}}'.format(MD_VERSION_STR)
 COMMIT_STATUS_FORMAT = '{0} {{0}}'.format(VERSION_STR)
 
+FARCY_COMMENT_START = '_{}_'.format(MD_VERSION_STR)
+
+
+def extract_issues(text):
+    """Extract farcy violations from a text."""
+    if not is_farcy_comment(text):
+        return []
+    # Strip out start of bullet point, ignore first line
+    return [line[2:] for line in text.split('\n')[1:]]
+
+
+def filter_comments_from_farcy(comments):
+    """Filter comments for farcy comments."""
+    return (comment for comment in comments if is_farcy_comment(comment.body))
+
+
+def filter_comments_by_path(comments, path):
+    """Filter a comments iterable by a file path."""
+    return (comment for comment in comments if comment.path == path)
+
+
+def is_farcy_comment(text):
+    """Return boolean if text was generated by Farcy."""
+    return text.startswith(FARCY_COMMENT_START)
```

Comment:
Do we want to only match if the version number matches? Seems like we should ignore that part.